.junie/guidelines.md

```diff
@@ -0,0 +1,194 @@
+# Zaakafhandelcomponent Development Guidelines
+
+This document provides essential information for developers working on the Zaakafhandelcomponent project.
+
+## Build/Configuration Instructions
+
+### Prerequisites
+
+- JDK 21 (required by WildFly)
+- Node.js (version specified in gradle.properties)
+- Docker and Docker Compose (for running integration tests and local development)
+
+### Building the Project
+
+The project uses both Gradle and Maven for building:
+
+- **Gradle** is used for most build tasks, including compiling, testing, and generating client code.
+- **Maven** is used specifically for generating the WildFly bootable JAR.
+
+#### Basic Build Commands
+
+```bash
```

Comment:
should this not be `shell` ?
```suggestion
```shell
```

Reply:
Hmm, I guess it could be. This entire file was generated by Junie so far. However we can now edit it to give Junie more help. Also nice is that you can add here technology which you do NOT want Junie to suggest. :-)